dockers/docker-sonic-mgmt-framework/Dockerfile.j2

```diff
@@ -20,7 +20,7 @@ RUN pip install connexion==1.1.15 \
 
 ## Install redis-tools dependencies
 ## TODO: implicitly install dependencies
-RUN apt-get -y install libjemalloc1 libatomic1 liblua5.1-0 lua-bitop lua-cjson
+RUN apt-get -y install libjemalloc2 libatomic1 liblua5.1-0 lua-bitop lua-cjson
```

Comment:
you can remove since it is in the base-buster image.

Reply:
Will update